eahub/localgroups/admin.py

```diff
@@ -45,41 +62,104 @@ class Meta:
             "other_info",
         ]
 
+    @classmethod
+    def widget_from_django_field(cls, f, default=widgets.Widget):
+        if isinstance(f, ArrayField):
+            return EnumArrayWidget
+        else:
+            return super().widget_from_django_field(f)
+
+    def import_row(
+        self,
+        row,
+        instance_loader,
+        using_transactions=True,
+        dry_run=False,
+        raise_errors=False,
+        **kwargs,
+    ):
+        row = self.remove_chars_from_row_keys(row, "\ufeff")
```

Comment:
that's interesting, I'm not sure what it does, should we leave a comment?

Reply:
has to do with UTF encoding. The csv comes in with this char for the first columnname, so "id" is "\ufeffid" instead. I tried changing the encoding configuration as suggested [here](https://stackoverflow.com/a/53511822), but it didn't work, so I used this approach. I'll rename the method to make it clearer